### ISSUE TITLE
Cleanup the CSS

### ISSUE THREAD

**Issue by IAmSurajBobade** [IAmSurajBobade/IAmSurajBobade.github.io#25]
At the movement, CSS works as expected for the webpage when loaded with JS supported/enabled browser.
However, no standard practice is followed, with this issue idea is to update the code to use best practices.

**Comment by krissh-codes** [comment]
Do you really need some help are just labelling it for fun?? 🤔

**Comment by IAmSurajBobade** [comment]
> Do you really need some help are just labelling it for fun?? 🤔

Hi @KrishnaMoorthy12 , thanks for stopping by. Yes, I need some help. If you think it's fun you can skip it.

**Comment by krissh-codes** [comment]
I went through the code.. there was not much going on.. that's why I asked if you really needed some help. I thought you might be experimenting with the Projects feature in GitHub.

**Comment by IAmSurajBobade** [comment]
You are right @KrishnaMoorthy12 , I am not a full-time web-developer. The profile readme also mentions it.
I have just started out of curiosity. Talking about the content I have some improvement ideas in the pipeline, which I plan to add step-by-step. However, I want to follow the best practices on the way. That is all. Also, I really appreciate your curiosity.
